octoprint_automation_scripts/__init__.py

```diff
@@ -332,12 +379,15 @@ def get_assets(self):
 
     def get_api_commands(self):
         return dict(
-            [(script_id, []) for script_id in self.scripts]
+            [(script_id, []) for script_id in self.scripts] +
+            [('cancel', [])]
         )
 
     def on_api_command(self, command, data):
         if command in self.scripts:
             self.start(command)
+        elif command == 'cancel':
+            self.relinquish_control(wait=False)
```

Comment:
I noticed that if there's a script with id "cancel", this will never get triggered.  Maybe flip these to defend against that.  But to really fix it, can we use some kind of reserved character prefix that couldn't appear in the script id?